pkg/types/types.go

```diff
@@ -29,18 +29,18 @@ func Init(nodeName string) {
 
 type CommonData struct {
 	// Node where the event comes from
-	Node string `json:"node,omitempty"`
+	Node string `json:"node,omitempty" column:"node,width:16" columnTags:"kubernetes"`
 
 	// Pod namespace where the event comes from, or empty for host-level
 	// event
-	Namespace string `json:"namespace,omitempty"`
+	Namespace string `json:"namespace,omitempty" column:"namespace,width:16" columnTags:"kubernetes"`
 
 	// Pod where the event comes from, or empty for host-level event
-	Pod string `json:"pod,omitempty"`
+	Pod string `json:"pod,omitempty" column:"pod,width:30" columnTags:"kubernetes"`
 
 	// Container where the event comes from, or empty for host-level or
 	// pod-level event
-	Container string `json:"container,omitempty"`
+	Container string `json:"container,omitempty" column:"container,width:25" columnTags:"kubernetes,runtime"`
```

Comment:
Do (K8s) containers usually have such a long name?

Reply:
I have two examples: `calico-kube-controllers` and `kube-controller-manager`, which seem to me quite common. WDYT?